### PR TITLE
Updates to documentation

### PR DESCRIPTION
**Pull request by mshadbolt**
## Summary of changes:
- Added more detail to how to add `candig.docker.internal` to the hosts 
- added to the install and ingest docs
- removed template Dockerfile as it was out of date and updated the README in templates to refer to the new confluence page and repository template

### CODE REVIEW

**Comment by mshadbolt** [comment]
Would be great if you can double check the command for getting local ip works on linux

```
# on linux
ifconfig eth0 | awk '/inet addr/ {gsub("addr:", "", $2); print $2}'
```

**Comment by stobbsm** [comment]
> Would be great if you can double check the command for getting local ip works on linux
> 
> ```
> # on linux
> ifconfig eth0 | awk '/inet addr/ {gsub("addr:", "", $2); print $2}'
> ```

This doesn't work on many modern distros due to the naming scheme for devices being different, with predicable names. Basically, the vast majority of distros don't use the ethX naming scheme anymore, in favour of naming things predictably every time.

Usually those names start with `en` for a physical network device. I'd suggest limiting the search to devices that start with that on Linux.

For reference, Redhat knowledge base about predictable naming: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-understanding_the_predictable_network_interface_device_names

EDIT: Can I suggest looking for the IP to use based on the default route instead?
`ip route | awk -F\  '/default/ {print $9}'` as an example. I'm sure there are better ways.

**Comment by mshadbolt** [comment]
Thanks for the feedback @stobbsm! 

Out of curiosity, are you associated with one of the sites trying to set up a CanDIG node for MoHCCN data or just generally interested in CanDIG?

**Comment by stobbsm** [comment]
> Thanks for the feedback @stobbsm!
> 
> Out of curiosity, are you associated with one of the sites trying to set up a CanDIG node for MoHCCN data or just generally interested in CanDIG?

I was hired on by the University of Calgary, so hopefully I’ll be making a lot of contributions where I’m able. Looking forward to collaborating.

**Comment by mshadbolt** [comment]
need to add the new issue that Francis raised here: https://candig.slack.com/archives/C01030VMCUR/p1705005688419419

**Review by SonQBChau — COMMENTED** [review]
I updated Mac M1 section

File: docs/ingest-and-test.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ The easiest way to test your local installation is to run the integration tests.
 make test-integration
 ```
 
+If you get an error about missing python requirements such as `dotenv` ensure the `candig` conda environment is activated, it should be in brackets beside your username, if it isn't run
```
Comment (daisieh) [inline docs/ingest-and-test.md]:
```suggestion
If you get an error about missing python requirements such as `dotenv` ensure the `candig` conda environment is activated, it should be in brackets beside your username. If it isn't run
```

File: docs/ingest-and-test.md [hunk 2]
```diff
@@ -29,55 +41,40 @@ curl -X "POST" "http://candig.docker.internal:8080/auth/realms/candig/protocol/o
...
-https://github.com/CanDIG/candigv2-ingest.git
+```commandline
+FAILED etc/tests/test_integration.py::test_server_count - assert 0 > 0
+FAILED etc/tests/test_integration.py::test_services_count - assert 0 > 0
+FAILED etc/tests/test_integration.py::test_federation_call - AssertionError: assert 'results' in {'error': 'There was a problem proxying the request'}
+FAILED etc/tests/test_integration.py::test_add_server - IndexError: list index out of range
 ```
 
-Create a virtual environment named `.venv`:
+You might need to rebuild federation service. First check whether `federation` is in the list of `CANDIG_AUTH_MODULES` in your `.env` file, if it isn't add it.
```
Comment (daisieh) [inline docs/ingest-and-test.md]:
```suggestion
You might need to rebuild federation service. First check whether `federation` is in the list of `CANDIG_AUTH_MODULES` in your `.env` file. If it isn't, add it.
```

File: docs/ingest-and-test.md [hunk 3]
```diff
@@ -29,55 +41,40 @@ curl -X "POST" "http://candig.docker.internal:8080/auth/realms/candig/protocol/o
...
+```commandline
 python settings.py
 source env.sh
+cd lib/candig-ingest/candigv2-ingest
+# should be pip install -r requirements.txt, but that didn't seem to work last I checked -- dependency errors?
+pip install -r requirements.txt
+python katsu_ingest.py --input tests/clinical_ingest.json
+
 ```
+You should then be able to visit the data portal and ingested data.
```
Comment (daisieh) [inline docs/ingest-and-test.md]:
```suggestion
You should then be able to visit the data portal and view ingested data.
```

**Review by daisieh — APPROVED** [review]
I made tiny grammatical suggestions; otherwise this looks great.

**Pull request merged**